src/bot.js

```diff
@@ -132,6 +132,31 @@ function replyToMessages() {
       network.blockAuthorOfMessageWithId(message['id']);
     });
 
+  filterPrivateMessages(messages)
+    .filter(message => message['subject'].match(/refresh (\w+)/i))
+    .forEach(message => {
+      const commentId = message['subject'].match(/refresh (\w+)/i)[1];
+      const comment = network.getComment('t1_' + commentId);
+
+      if (! comment) return;
```

Comment:
This is really just a style thing, but I prefer if statements always being written like
```
if (! comment) {
  return;
}
```
I feel like always having the curly braces & always adding an endline (even on a one-line if) makes it more obvious where the if statement starts and ends, and this scoping prevents bugs. Open to discussion though

Reply:
Sure, I agree that there are advantages to being explicit with short statements. I will change the style of my return statements.